rust/src/delta_arrow.rs

```diff
@@ -85,7 +85,8 @@ impl TryFrom<&schema::SchemaDataType> for ArrowDataType {
                     "float" => Ok(ArrowDataType::Float32),
                     "double" => Ok(ArrowDataType::Float64),
                     "boolean" => Ok(ArrowDataType::Boolean),
-                    "binary" => Ok(ArrowDataType::Binary),
+                    // Change to binary type after https://github.com/apache/arrow-rs/pull/702 is released
+                    "binary" => Ok(ArrowDataType::Utf8),
```

Comment:
Your PR has already been merged and released. We are now pinning to the latest arrow master branch, so we can use your upstream arrow change here now :)

Reply:
It seems that the snapshot version we are using doesn't include changes in my PR?
See: https://github.com/apache/arrow-rs/blob/fa5acd971c973161f17e69d5c6b50d6e77c7da03/arrow/src/json/reader.rs

Reply:
good call, can you update it to the latest commit or to your commit? I am fine with either way.

Reply:
Okay, I've updated the version to the latest commit.